setup.py

```diff
@@ -74,6 +46,7 @@ def run_tests(self):
         'Programming Language :: Python',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.6',
```

Comment:
Shall we add Django also here?

Reply:
classifiers are not mandatory and only useful for "real" public project. we can opt to add Django but what about others? https://pypi.org/classifiers/

Reply:
at least we should have Development Status, Python, Django